lib/services/client_data_creator.rb

```diff
@@ -0,0 +1,41 @@
+class ClientDataCreator
+  def initialize(client_data, user, attachment_data = [])
+    @client_data = client_data
+    @user = user
+    @attachment_data = attachment_data
+  end
+
+  def run
+    client_data.build_proposal(flow: 'linear', requester: user)
```

Comment:
happens here: https://github.com/18F/C2/blob/master/app/controllers/use_case_controller.rb#L73 (this controller is shared for NCR work orders and 18F procurements)